### PR TITLE
Backport CVE patch for JetPack

### PR DESCRIPTION
**Pull request by fpetrini15**
Backport CVE patch for JetPack
Core changes: https://github.com/triton-inference-server/core/pull/302

### CODE REVIEW

File: Dockerfile.win10.min [hunk 1]
```diff
@@ -76,7 +76,7 @@ RUN git clone --single-branch --depth=1 -b %VCPGK_VERSION% https://github.com/mi
 WORKDIR /vcpkg
 RUN bootstrap-vcpkg.bat
 RUN vcpkg.exe update
-RUN vcpkg.exe install openssl:x64-windows openssl-windows:x64-windows rapidjson:x64-windows re2:x64-windows boost-interprocess:x64-windows boost-stacktrace:x64-windows zlib:x64-windows pthread:x64-windows b64:x64-windows
+RUN vcpkg.exe install openssl:x64-windows openssl-windows:x64-windows rapidjson:x64-windows re2:x64-windows boost-filesystem:x64-windows boost-interprocess:x64-windows boost-stacktrace:x64-windows zlib:x64-windows pthread:x64-windows b64:x64-windows
```
Comment (mc-nv) [inline Dockerfile.win10.min:79]: We won't be able to rebuild the container using this change most likely.

Comment (fpetrini15) [reply Dockerfile.win10.min:79]: I'll run a release pipeline to check.

Comment (mc-nv) [reply Dockerfile.win10.min:79]: It's a manual action that required to happen before the pipeline, action here is to rebuild the windows base container image.

**Comment by rmccorm4** [comment]
Doing some cleanup - Is this PR still needed?

**Pull request closed without merging**